package.json

```diff
@@ -15,6 +15,7 @@
     "@strapi/plugin-users-permissions": "4.1.9",
     "@strapi/strapi": "4.1.9",
     "pg": "^8.7.3",
+    "qs": "^6.10.3",
```

Comment:
You've installed this but you're not importing it into any other file. Is it required by strapi? If it's not, can you remove it?

Reply:
actually I think it will use later, but I haven't implemented yet. but for now, ok I'll remove it. Thanks cak